frontend/packages/dev-console/src/components/topology/topology-types.ts

```diff
@@ -27,6 +27,7 @@ export interface TopologyDataResources {
   pipelines?: FirehoseResult;
   pipelineRuns?: FirehoseResult;
   eventSourceCronjob?: FirehoseResult;
+  pingsource?: FirehoseResult;
```

Comment:
@invincibleJai it would be good if we started using `referenceForModel(PingsourceModel)`. With my topology PR these key wont exist anymore.
I will change the other keys in my PR to use the same.